cmd/root.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/morikuni/aec"
+	"github.com/openfaas-incubator/ofc-bootstrap/version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Version as per git repo
+	Version string
+
+	// GitCommit as per git repo
+	GitCommit string
+)
+
+// WelcomeMessage to introduce ofc-bootstrap
+const WelcomeMessage = "Welcome to ofc-bootstrap! Find out more at https://github.com/openfaas-incubator/ofc-bootstrap"
+
+func init() {
+	rootCommand.AddCommand(versionCmd)
+	rootCommand.Flags().StringArrayP("yaml", "f", []string{""}, "The init.yaml plan file")
```

Comment:
is there a way to exclude this from the "usage" output? Its showing as a command and I though it was deprecated?

<img width="402" alt="Screenshot 2019-12-04 at 07 59 12" src="https://user-images.githubusercontent.com/40488132/70124279-58c7f580-166c-11ea-90f8-f91b0442dece.png">

Reply:
Which usage do you want to exclude? How did you get it to show?

Reply:
running ./ofc-bootstrap creates the above screenshot output, --yaml has been depricated for --file, or -f?

Reply:
For `ofc-bootstrap` on its own I think that is OK

`ofc-bootstrap --yaml file.yaml` -> produces the warning.